webhook_server/tests/e2e/helpers.py

```
@@ -0,0 +1,427 @@
+        test_repo: Test repository in format owner/repo-name
+
+    Returns:
+        list[dict[str, Any]]: List of linked issue dictionaries
+
+    Raises:
+        subprocess.CalledProcessError: If API call fails
+    """
+    logger.debug(f"Getting {test_repo} issues")
+
+    # Get all issues in the repository
+    result = subprocess.run(
```

Ruff (0.14.5)

`subprocess` call: check for execution of untrusted input

(S603)